MachObfuscator/DependencyAnalysis/ObfuscationPaths+Building.swift

```diff
@@ -1,13 +1,14 @@
 import Foundation
 
 extension ObfuscationPaths {
-    static func forAllExecutablesWithDependencies(inDirectory dir: URL, fileRepository: FileRepository = FileManager.default, dependencyNodeLoader: DependencyNodeLoader) -> ObfuscationPaths {
+    static func forAllExecutablesWithDependencies(inDirectory dir: URL, fileRepository: FileRepository = FileManager.default, dependencyNodeLoader: DependencyNodeLoader, obfuscableFilesFilter: ObfuscableFilesFilter) -> ObfuscationPaths {
         var paths = ObfuscationPaths()
-        // TODO: create better exclusion mechanism (with custom excludes and includes)
+        
+        // TODO: create better exclusion mechanism (with custom excludes and includes)    
```

Comment:
This TODO can probably be removed now :)

Reply:
done